activesupport/lib/active_support/core_ext/numeric/time.rb

```diff
@@ -5,6 +5,11 @@
 require 'active_support/core_ext/date/acts_like'
 
 class Numeric
+  # Convert number to local Time.
+  def to_time
+    Time.at(self)
```

Comment:
Yeah, I was afraid of this. Shall I throw away those commits and stick with the implementation up to ec2e09b9dcc850d816685dc16231f026f350e612?